tests/Functional/Action/WebHook/UpdateLineItemsWebhookActionTest.php

```diff
@@ -68,6 +68,37 @@ protected function setUp(): void
         $this->setUpTestLogHandler();
     }
 
+    public function testItReturns401IfNotAuthenticated(): void
+    {
+        $this->kernelBrowser->request(
+            Request::METHOD_POST,
+            '/api/v1/web-hooks/update-line-items',
+            [],
+            [],
+            [],
+            ''
+        );
+
+        self::assertEquals(Response::HTTP_UNAUTHORIZED, $this->kernelBrowser->getResponse()->getStatusCode());
+    }
+
+    public function testItReturns401IfWrongAuthentication(): void
```

Comment:
Proposal for a case name: `testItReturns401IfWrongCredentialsPassed`

Reply:
Thanks, fixed